### ISSUE TITLE
Log which circuits failed during scan.

### ISSUE THREAD

**Issue by NullHypothesis** [NullHypothesis/exitmap#25]
We should log the exit relays for which we were unable to create circuits during a scan. Perhaps this information should even be logged to disk, so it's easier to later find relays for which circuits failed repeatedly.

**Comment by s4chin** [comment]
I want to try this one. How do I go about this?

**Comment by NullHypothesis** [comment]
When a circuit to an exit relay cannot be created, it raises an exception here: https://github.com/NullHypothesis/exitmap/blob/master/src/exitmap.py#L394 I would keep track of all failed circuits, and print them once exitmap is finished.

**Comment by frankcash** [comment]
Hello,  it's been a while since I've helped work on this project.  I'd love to contribute another PR.  Is there already a function that runs at the end of the "mapping" process or would one need to write this?
